src/wombatLite.js

```diff
@@ -13,6 +13,7 @@ export default function WombatLite($wbwindow, wbinfo) {
   this.wb_info.top_host = this.wb_info.top_host || '*';
   this.wb_info.wombat_opts = this.wb_info.wombat_opts || {};
   this.WBAutoFetchWorker = null;
+  this.historyCB = [];
```

Comment:
If we do not expect to be using multiple callbacks it would be simplest to work with one
```suggestion
  this.historyCB = null;
```